### ISSUE TITLE
Windows installers needs Authenticode signature for Zap 2.6.0

### ISSUE THREAD

**Issue by jtcmedia** [zaproxy/zaproxy#3337]
Trying to install the new 2.6.0 on Windows 10 brings up the Windows SmartScreen warning.

![capture](https://cloud.githubusercontent.com/assets/3260027/24470829/1d8e0b2c-1475-11e7-8f74-a6fecf3a73f2.PNG)

**Comment by psiinon** [comment]
Yes, we know about this - none of the previous ZAP Windows releases were signed either.
We are working on it but getting a code signing certificate is taking longer than anticipated.
